container-toolkit/index.md

```diff
@@ -23,16 +23,19 @@ Container Device Interface <cdi-support.md>
 docker-specialized.md
 ```
 
-The NVIDIA Container Toolkit enables users to build and run GPU-accelerated containers.
-The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container)
-and utilities to automatically configure containers to leverage NVIDIA GPUs.
+The NVIDIA Container Toolkit is a collection of libraries and utilities enabling users to build and run GPU-accelerated containers.
 
-```{image} https://cloud.githubusercontent.com/assets/3028125/12213714/5b208976-b632-11e5-8406-38d379ec46aa.png
-:width: 400
```

Comment:
I propose removing the diagram. It might be a provocative change. My main argument is that the diagram does not help answering what the toolkit is and where it fits the big picture. In particular, it does not mention: the container toolkit, the runtime library, i.e. it does not really help clarify what the toolkit is and does. It however mentions a different toolkit, and that might be confusing.

That's the diagram: 
![image](https://github.com/user-attachments/assets/770d7e25-302b-4c88-a030-4bf4b4f7c603)